django_distill/renderer.py

```diff
@@ -179,19 +179,19 @@ def render_file(self, view_name, status_codes, view_args, view_kwargs):
             uri = self.generate_uri(url, view_name, view_args)
             args = view_args
         render = self.render_view(uri, status_codes, args, a)
-        file_name = self._get_filename(file_name, uri)
+        file_name = self._get_filename(file_name, uri, args)
         return uri, file_name, render
     
     def render_all_urls(self):
         for url, distill_func, file_name_base, status_codes, view_name, a, k in self.urls_to_distill:
             for param_set in self.get_uri_values(distill_func, view_name):
-                if not param_set:
+                if param_set is None:
```

Comment:
`param_set` here can be an empty tuple, depending on some Django internals and versions. `get_uri_values` mirrors this so changing this to `is None` prevents it from catching an empty `()` `param_set`. This probably needs to remain as `if not`.

Reply:
the line this condition is guarding just sets it to an empty tuple, so if it's already an empty tuple there's nothing to do

Reply:
unless we need to coerce other types like empty list, empty string to empty tuple too?

happy to change it back if you wish

Reply:
Yes, it can be an empty list in some cases. It's probably best to leave it as a generic falsy catch-all. See: https://github.com/meeb/django-distill/blob/66830a9fbf9c90f713f2ee34c242ac78b79b0ded/django_distill/renderer.py#L235